File: framework/ecommerce-framework-dxa-module/src/main/java/com/sdl/ecommerce/dxa/model/SearchFeedbackWidget.java
package com.sdl.ecommerce.dxa.model;


import com.sdl.ecommerce.api.model.QuerySuggestion;
import com.sdl.webapp.common.api.mapping.annotations.SemanticEntity;
import com.sdl.webapp.common.api.mapping.annotations.SemanticProperty;
import com.sdl.webapp.common.api.model.RichText;
import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;

import java.util.List;

import static com.sdl.webapp.common.api.mapping.config.SemanticVocabulary.SDL_CORE;

/**
 * Search Feedback Widget
 *
 * @author nic
 */
@SemanticEntity(entityName = "SearchFeedback", vocabulary = SDL_CORE, prefix = "e", public_ = false)
public class SearchFeedbackWidget extends AbstractEntityModel {

    @SemanticProperty("e:spellCheckLabel")
    private RichText spellCheckLabel;


    private List<QuerySuggestion> querySuggestions;

    public List<QuerySuggestion> getQuerySuggestions() {
        return querySuggestions;
    }

    public void setQuerySuggestions(List<QuerySuggestion> querySuggestions) {
        this.querySuggestions = querySuggestions;
    }

    public RichText getSpellCheckLabel() {
        return spellCheckLabel;
    }

}

File: dxa.net/ecommerce-framework-dotnet/SDL.ECommerce.DXA/Models/SearchFeedbackWidget.cs
using Sdl.Web.Common.Models;
using SDL.ECommerce.Api.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace SDL.ECommerce.DXA
{
    /// <summary>
    /// Search Feedback Widget
    /// </summary>
    [SemanticEntity(Vocab = CoreVocabulary, EntityName = "SearchFeedbackWidget", Prefix = "e")]
    public class SearchFeedbackWidget : EntityModel
    {
        [SemanticProperty("e:spellCheckLabel")]
        public RichText SpellCheckLabel { get; set; }

        [SemanticProperty(IgnoreMapping = true)]
        public IList<IQuerySuggestion> QuerySuggestions { get; set; } 
    }
}